packages/vscode-dependencies-validation/package.json

```diff
@@ -0,0 +1,54 @@
+{
+  "name": "@sap-devx/vscode-dependencies-validation",
```

Comment:
i do not believe vscode extension names can include scopes (@sap-devx)

Reply:
Done